source/retryable-reads/tests/changeStreams-client.watch-serverErrors.yml

```diff
@@ -6,15 +6,12 @@ data:
     - {_id: 2, x: 22}
     - {_id: 3, x: 33}
 
-minServerVersion: '4.1.7' # 4.2
+minServerVersion: '4.0'
+topology: [replicaset, sharded]
 
 tests:
     -
-        description: "client.watch succeeds after InterruptedAtShutdown"
-        topology:  [replicaset, sharded]
-        clientOptions:
-            retryReads: true
-        failPoint: &failCommand_failPoint
+        description: "client.watch succeeds after InterruptedAtShutdown"        failPoint: &failCommand_failPoint
```

Comment:
The format got messed up here. Notice how `description` and `failPoint` are on the same line. I think it's just in this file.

Reply:
Thanks, fixed!

Reply:
Can you fix the same issue with the rest of the tests in this file?

Reply:
Gah, I'm really sorry! Fixed for real this time.